kubernetes/services/verifier.yml.j2

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: verifier
+spec:
+  selector:
+    matchLabels:
+      run: verifier
+  serviceName: "verifier"
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        run: verifier
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+        - name:             concent-api
+          image:            {{ docker_registry }}/{{image_prefix }}verifier:{{ image_version }}
+          imagePullPolicy:  Always
+          resources:
+            requests:
+              memory: "1Gi"
+              cpu:    "600m"
+            limits:
```

Comment:
Yes, I think that these resources are too low, but from what I remember, we set them together. For more resources we need another node.

Reply:
OK. Let's leave it as is for now in that case. I'll need to get back to this later, in a more generic way so that we can have different requirements for different clusters.